data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -350,12 +350,14 @@ then
 			--dest-dir=/assets/cco-bootstrap \
 			--manifests-dir=/assets/manifests \
 			--cloud-credential-operator-image=${CLOUD_CREDENTIAL_OPERATOR_IMAGE}
+			{{- if .CloudCredentialsMode}} \
+			--force-mode={{.CloudCredentialsMode}}
+			{{- end}}
 
 	cp cco-bootstrap/manifests/* manifests/
 	# skip copy if static pod manifest does not exist (ie CCO has been disabled)
-	if [ -f cco-bootstrap/bootstrap-manifests/cloud-credential-operator-pod.yaml ]; then
-		cp cco-bootstrap/bootstrap-manifests/* bootstrap-manifests/
-	fi
+	[ -d cco-bootstrap/bootstrap-manifests ] && [ "$(ls -A cco-bootstrap/bootstrap-manifests)" ] && \
+	  cp cco-bootstrap/bootstrap-manifests/* bootstrap-manifests/
```

Comment:
this is not very readable imo, try and use if block.

Also please include the reasoning as to why `ls` condition is being added.